src/components/table/table-row/table-row.style.js

```diff
@@ -0,0 +1,200 @@
+import { css } from 'styled-components';
+import StyledTableCell from '../table-cell/table-cell.style';
+import StyledTableHeader from '../table-header/table-header.style';
+import { isClassic } from '../../../utils/helpers/style-helper';
+import { THEMES } from '../../../style/themes';
+
+const StyledTableRow = css`
+  .carbon-table-row {
```

Comment:
i think that class should be rewritten to a styled-component

Reply:
I've added a comment to the top of the file, the current version of React-DnD used by the DragAndDrop component doesn't support styled-components, so I had to go with the the current implementation until that can be fixed as part of another ticket